pom.xml

```diff
@@ -6,7 +6,7 @@
   <artifactId>amazon-kinesis-client</artifactId>
   <packaging>jar</packaging>
   <name>Amazon Kinesis Client Library for Java</name>
-  <version>1.11.3-SNAPSHOT</version>
+  <version>1.11.3</version>
```

Comment:
We need to change this as part of the release PR, and not as part of the standard PR

Reply:
Ok, change it back for now